src/templates/en/2021/methodology.html

```diff
@@ -0,0 +1,474 @@
+      </p>
+
+      <p>
+        The results of each test are made available as a <a href="https://en.wikipedia.org/wiki/HAR_(file_format)">HAR file</a>, a JSON-formatted archive file containing metadata about the web page.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="lighthouse"><a href="#lighthouse" class="anchor-link">Lighthouse</a></h3>
+
+      <p>
+        <a hreflang="en" href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a> is an automated website quality assurance tool built by Google. It audits web pages to make sure they don&#8217;t include user experience antipatterns like unoptimized images and inaccessible content.
+      </p>
+
+      <p>
+        HTTP Archive runs the latest version of Lighthouse for all of its mobile web pages — desktop pages are not included because of limited resources. As of the July {{ year }} crawl, HTTP Archive used a combination of <a hreflang="en" href="https://github.com/GoogleChrome/lighthouse/releases/tag/v8.0.0">8.0.0</a> anmd <a hreflang="en" href="https://github.com/GoogleChrome/lighthouse/releases/tag/v8.1.0">8.1.0</a> versions of Lighthouse.
+      </p>
+
+      <p>
+        Lighthouse is run as its own distinct test from within <a href="#webpagetest">WebPageTest</a>, but it has its own configuration profile:
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+            <th>Config</th>
+            <th>Value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>CPU slowdown</td>
+            <td>1x/4x</td>
+          </tr>
+          <tr>
+            <td>Download throughput</td>
+            <td>1.6 Mbps</td>
+          </tr>
+          <tr>
+            <td>Upload throughput</td>
+            <td>0.768 Mbps</td>
+          </tr>
+          <tr>
+            <td>RTT</td>
+            <td>150 ms</td>
+          </tr>
+        </tbody>
+      </table>
+      </div></div>
+
+      <p>
+        For more information about Lighthouse and the audits available in HTTP Archive, refer to the <a hreflang="en" href="https://developers.google.com/web/tools/lighthouse/">Lighthouse developer documentation</a>.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="wappalyzer"><a href="#wappalyzer" class="anchor-link">Wappalyzer</a></h3>
+
+      <p>
+        <a hreflang="en" href="https://www.wappalyzer.com/">Wappalyzer</a> is a tool for detecting technologies used by web pages. There are <a hreflang="en" href="https://www.wappalyzer.com/technologies">90 categories</a> of technologies tested, ranging from JavaScript frameworks, to CMS platforms, and even cryptocurrency miners. There are over 2,600 supported technologies (and increase from 1,400 last year).
```

Comment:
Updated the numbers but never took a note what they were in July and think they've increased a lot. But think it's OK. What do you think @rviscomi ?

Reply:
LGTM